src/mount/readdata_cache.h

```diff
@@ -424,13 +437,16 @@ class ReadCache {
 	}
 
 	void clearReserved(unsigned count) {
-		std::unique_lock<std::mutex> lock(gUsedReadCacheMemoryMutex,
+		std::unique_lock<std::mutex> lock(gReadCacheMemoryMutex,
 		                                  std::defer_lock);
 		while (!reserved_entries_.empty() && count-- > 0) {
 			Entry *e = std::addressof(reserved_entries_.front());
 			if (e->refcount == 0) {
 				lock.lock();
 				gUsedReadCacheMemory -= e->buffer.size();
+				gReadCacheMemoryAlmostExceeded =
+				    gUsedReadCacheMemory >=
+				    static_cast<uint64_t>(0.8 * gReadCacheMaxSize.load());
```

Comment:
Same here, or even better, an inline funtion, maybe named updateReadCacheMemoryAlmostExceeded, and reuse it everywhere.

Reply:
Done